app/controllers/pay/payments_controller.rb

```diff
@@ -1,6 +1,7 @@
 module Pay
   class PaymentsController < ApplicationController
     def show
+      @redirect_to = params[:back] || root_path
```

Comment:
We should probably change this to `params[:back].presence || root_path` so that an empty string is handled as nil.

Reply:
Ahh very nice. Wasn't aware of `.presence`